source/style-guide/information-types/prototypes/reference/server-params-style.txt

```diff
@@ -0,0 +1,129 @@
+.. _meta-reference-server-params:
+
+=======================
+Server Parameters Style
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Details to Document
+-------------------
+
+When documenting a new server parameter, include the following details 
+in addition to specific information about the new parameter:
+
+.. list-table::
+   :widths: 40 40
+   :header-rows: 1
+
+   * - Documentation Detail
```

Comment:
[Question]

Are these details listed in the order that they should appear on the docs page? If so, can we make that clear?

Reply:
Updated. Thanks!